### DISCUSSION TITLE
Shorter alias generator (8 random characters)

### DISCUSSION THREAD

**Discussion by kelszo** [simple-login/app#1212] [Feature Requests]
It would be lovely to have the ability to generate shorter random aliases. Both UUID and random words generate rather long aliases. So an ability to generate aliases just based on 8 more random characters would be nice.

Example:
yvs31q2k@8shield.net
vs
fd0bd0bc-eef6-4cd5-bd6a-d6c7c3a6a2e7@8shield.net

**Comment by Nautman** [comment]
I agree!

**Comment by Nautman** [comment]
Seems like this PR (#996) and issue (#1252) is on the same topic.
